boot/Makefile

```diff
@@ -0,0 +1,10 @@
+LIBDIR ?= /usr/lib
+SYSCONFDIR ?= /etc
+GRUBCONFDIR ?= $(SYSCONFDIR)/grub.d
+DRACUTCONFDIR ?= $(LIBDIR)/dracut/dracut.conf.d
+
+.PHONY: install
+
+install:
+	install -D -m 0644 grub.qubes $(DESTDIR)$(GRUBCONFDIR)/30_qubes
```

Comment:
This needs to go into `/etc/default/grub.d` on Debian. And somewhere similar (but to be included manually) on Fedora.

Reply:
Please see `debian/rules`.